README.md

```diff
@@ -511,14 +555,15 @@ configure-loopback:
 ```
 
 The remove-loopback target will be used so that you can execute the test multiple times - it makes sure that loopback service code is always executed.
```

Comment:
can you add backsticks to "remove-loopback"? so it has the same look as the other targets.